packages/scandipwa/src/component/MyAccountCustomerForm/MyAccountCustomerForm.component.js

```diff
@@ -31,7 +31,8 @@ export class MyAccountCustomerForm extends FieldForm {
         handlePasswordInput: PropTypes.func.isRequired,
         email: PropTypes.string,
         currentPassword: PropTypes.string,
-        vatNumberRequired: PropTypes.bool.isRequired
+        vatNumberRequired: PropTypes.bool.isRequired,
+        range: PropTypes.objectOf(PropTypes.number).isRequired
```

Comment:
Maybe using type `PropTypes.shape({ min: PropTypes.number, max: PropTypes.number }).isRequired` will be more appropriate here?

Reply:
Done